extensions/standard-processors/processors/NetworkListenerProcessor.cpp

```diff
@@ -37,7 +38,8 @@ void NetworkListenerProcessor::onTrigger(const std::shared_ptr<core::ProcessCont
 }
 
 void NetworkListenerProcessor::startServer(
```

Comment:
Based on our conversation I think we should split this to startTCP/UDPServer and remove the arguments and create virtual functions to get max_queue_size, port, etc.

Reply:
Updated in d0e2ac6c3abeff8d46f8e15fd80816dd86fa9b13